test/e2e/autoscaling_utils.go

```diff
@@ -142,7 +143,6 @@ func (rc *ResourceConsumer) CleanUp() {
 	rc.stop <- 0
 	expectNoError(DeleteRC(rc.framework.Client, rc.framework.Namespace.Name, rc.name))
 	expectNoError(rc.framework.Client.Services(rc.framework.Namespace.Name).Delete(rc.name))
```

Comment:
The service may consume resource so you may want to stop the consumption explicitly.